package.json

```diff
@@ -47,7 +47,6 @@
     "mobify-code-style": "2.8.1",
     "mocha": "~2.3.4",
     "rollup": "^0.49.3",
-    "rollup-plugin-babel": "^3.0.2",
     "rollup-plugin-uglify": "^2.0.1",
```

Comment:
We can drop the uglify plugin too, then, right?

Reply:
👍